src/Request/FindPointsRequestBuilder.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\InpostBundle\Request;
+
+use Answear\InpostBundle\Enum\PointFunctionsType;
+use Answear\InpostBundle\Enum\PointType;
+
+class FindPointsRequestBuilder
+{
+    private array $criteria;
```

Comment:
all setters put something into this array. Only because you need to use it on build metod, is't it?
Why you don't create specific params `string name`, `PointType $type`...
and only build 'criteria' array on `build` metod?

Reply:
I don't think we want that array to have empty values, so in build method I would need to check each param to have value so we don't end up with something like "?name=&type=&city=" etc. Seems to me like more complex and unnecessary things to do instead of use array that user actually changed.

Reply:
You will need to check all properties to build. Ok, you may omit this comment.